### PR TITLE
Cleaning following the release of JabRef 3.3

### PR DESCRIPTION
**Pull request by mlep**

### CODE REVIEW

File: _faq/10_general.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ A: We are collecting all publications we hear about at <https://github.com/JabRe
 ## Q: JabRef does not start. What should I do?
 
 A: This may be because the preferences need to be reset.
-Execute `java -jar JabRef-3.2.jar --prdef all -n`.
+Execute `java -jar JabRef-3.3.jar --prdef all -n`.
```
Comment (simonharrer) [inline _faq/10_general.md]: I would use a more general pattern like JabRef-X.Y.jar

Comment (mlep) [inline _faq/10_general.md]: Changed! Thank you for your review.

File: _faq/10_general.md [hunk 2]
```diff
@@ -116,7 +115,7 @@ Files will automatically be linked with relative paths if the files are placed i
 ## Q: I want to export my bibliography entries into a simple text file so that I can import them into a spreadsheet easily. Is this possible?
 
 A: Yes. Use **File -&gt; Export**.
-Under “Filter:”, choose “OpenOffice CSV (\*.csv)”.
+Under “Filter:”, choose “OpenOffice\LibreOffice CSV (\*.csv)”.
```
Comment (oscargus) [inline _faq/10_general.md]: Forward slash (/)

Comment (mlep) [inline _faq/10_general.md]: Corrected! Thank you.

**Pull request merged**